src/JJMasterData.Core/WebComponents/JJDataExp.cs

```diff
@@ -212,14 +208,14 @@ public void DoExport(DataTable dt)
         exporter.CurrentContext = HttpContext.Current;
         exporter.AbsoluteUri = HttpContext.Current.Request.Url.AbsoluteUri;
         
-        exporter.RunWorkerAsync(CancellationToken.None).Wait();
+        Task.Run(async () => await exporter.RunWorkerAsync(CancellationToken.None));
```

Comment:
Using .Wait() is causes deadlocks
https://stackoverflow.com//questions/13140523/await-vs-task-wait-deadlock#answer-13140963

Reply:
Good Job!